### ISSUE TITLE
Fix common 404's in /web/tools

### ISSUE THREAD

**Issue by petele** [google/WebFundamentals#4023]
The following URLs have been requested at least 10 times in the last month and users got a 404 response. Please update the `_redirects.yaml` in `/web/tools/` to fix this.

* [x] /web/tools/chrome-devtools/evaluate-performance/network-performance/resource-loading
* [x] /web/tools/chrome-devtools/inspect-styles/basics
* [x] /web/tools/chrome-devtools/evaluate-performance/memory-problems/heap-snapshots
* [x] /web/tools/chrome-devtools/javascript/What do the colors mean in Chrome Developer Tools Scope panel
* [x] /web/tools/chrome-devtools/javascript/breakpoints/add-breakpoints?hl=en
* [x] /web/tools/chrome-devtools/javascript/console/
* [x] /web/tools/chrome-devtools/javascript/console/console-ui
* [x] /web/tools/chrome-devtools/javascript/breakpoints/
* [x] /web/tools/chrome-devtools/evaluate-performance/memory-problems/memory-101
* [x] /web/tools/chrome-devtools/javascript/command-line/command-line-reference
* [x] /web/tools/chrome-devtools/evaluate-performance/evaluate-performance/timeline-tool
* [x] /web/tools/chrome-devtools/evaluate-performance/evaluate-performance/timeline-tool?hl=en
* [x] /web/tools/chrome-devtools/javascript/console/console-write
* [x] /web/tools/lighthouse/audits/blocking-links
* [x] /web/tools/lighthouse/audits/has-viewport-meta-tag"
* [x] /web/tools/chrome-devtools/evaluate-performance/memory-problems/memory-101?hl=en
* [x] /web/tools/chrome-devtools/evaluate-performance/memory-problems/memory-diagnosis
* [x] /web/tools/chrome-devtools/evaluate-performance/rendering-tools/js-execution
* [ ] /web/tools/chrome-devtools/evaluate-performance/rail and https://crbug.com/574343
* [x] /web/tools/chrome-devtools/javascript/command-line/command-line-reference?hl=en
* [ ] /web/tools/chrome-devtools/network-performance/imgs/network-image-preview.png
* [ ] /web/tools/chrome-devtools/remote-debugging/imgs/remote-debug-banner.png
* [x] /web/tools/chrome-devtools/evaluate-performance/evaluate-performance/rail
* [x] /web/tools/chrome-devtools/evaluate-performance/evaluate-performance/rail?hl=en

**Comment by kaycebasques** [comment]
@petele are the following literal?

* `/web/tools/chrome-devtools/evaluate-performance/rail and https://crbug.com/574343` (extra text and URL at end)
* ` /web/tools/lighthouse/audits/has-viewport-meta-tag"` (extra double-quote at end)
* `/web/tools/chrome-devtools/javascript/What do the colors mean in Chrome Developer Tools Scope panel` (extra text at end)

**Comment by kaycebasques** [comment]
@petele

I tried setting up a redirect for `/web/tools/lighthouse/audits/has-viewport-meta-tag"` but it wasn't sticking... I don't know if I can fix that one. 

Same goes for `/web/tools/chrome-devtools/javascript/What do the colors mean in Chrome Developer Tools Scope panel`.

I searched all of WF for `/web/tools/chrome-devtools/network-performance/imgs/network-image-preview.png` and `/web/tools/chrome-devtools/remote-debugging/imgs/remote-debug-banner.png` (I actually just searched for `remote-debug-banner` and `network-image-preview` in case it was being referenced with relative paths) and it came up empty. That makes me think some external sites are referencing the images, in which case that doesn't really seem like our responsibility?
